app/controllers/warehouse_reports/health/eligibility_results_controller.rb

```diff
@@ -14,13 +14,28 @@ def show
         preload(:health_agency, :care_coordinator)
       @ineligible = patient_scope.where(medicaid_id: @inquiry.ineligible_ids).
         preload(:health_agency, :care_coordinator)
+      @aco_changes = patient_scope.where(medicaid_id: aco_changes).
+        preload(:health_agency, :care_coordinator)
     end
 
     def has_managed_care(patient) # rubocop:disable Naming/PredicateName
       @inquiry.managed_care_ids.include? patient.medicaid_id
     end
     helper_method :has_managed_care
 
+    def aco_changes
+      ids = []
+      referral_scope.where(medicaid_id: @inquiry.eligible_ids).each do |referral|
+        medicaid_id = referral.medicaid_id
+        edi_name = @inquiry.aco_names[medicaid_id]
+        if edi_name.present?
```

Comment:
don't you mean `next unless edi_name.present?`  Rubocop isn't going to like this.